server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -43,6 +45,8 @@
 import reactor.core.publisher.Mono;
 
 public class ObjectStorageBlobsDAO implements BlobStore {
+    private static final int READ_RETRIES_NUMBER = 5;
+    private static final Duration READ_RETRIES_BACKOFF = Duration.ofMillis(100);
```

Comment:
Shouldn't this be added to a configuration object?

Reply:
I do not thinks it worth it, I mean, if we need to support a less reliable BlobStore, maybe we will have to review our design.